tests/smoketests/test_phylogeneticdistance.py

```diff
@@ -0,0 +1,27 @@
+import tempfile
+
+from dendropy.calculate.phylogeneticdistance import PhylogeneticDistanceMatrix
+from dendropy.utility.container import DataTable
+
+from . import pytestmark
+
+
+def test_as_data_table():
+    mat = PhylogeneticDistanceMatrix()
+
+    res = mat.as_data_table()
+    assert isinstance(res, DataTable)
+
+
+def test_write_csv():
+    mat = PhylogeneticDistanceMatrix()
+    file = tempfile.NamedTemporaryFile()
```

Comment:
I would recommend using the context manager interface (i.e., `with tempfile.NamedFile as file:`). That way the file and the file handle will get cleaned up automatically.